### PR TITLE
cleanup

### PR DESCRIPTION
**Pull request by grosser**
@alexdunae @akzhan

### CODE REVIEW

File: premailer.gemspec [hunk 1]
```diff
@@ -1,23 +1,14 @@
...
   s.summary  = "Preflight for HTML e-mail."
   s.email    = "code@dunae.ca"
   s.homepage = "http://premailer.dialect.ca/"
   s.description = "Improve the rendering of HTML emails by making CSS inline, converting links and warning about unsupported code."
   s.has_rdoc = true
   s.author  = "Alex Dunae"
-  s.files            = `git ls-files`.split("\n")
-  s.test_files       = `git ls-files -- {test,spec,features}/*`.split("\n")
-  s.executables      = `git ls-files -- bin/*`.split("\n").map { |f| File.basename(f) }
+  s.files            = `git ls-files lib misc LICENSE.md README.md`.split("\n")
```
Comment (grosser) [inline premailer.gemspec]: smaller gem -> faster download + less space in git when vendoring

**Comment by Taher-Ghaleb** [comment]
Hello,

Sorry, I know it's been a long time ago, but I am curious why caching hasn't been enabled before this (i.e., since December 17, 2014, when it became available to open source projects).

Thank you.

**Comment by grosser** [comment]
idk what you are talking about, can you elaborate ?

**Comment by Taher-Ghaleb** [comment]
Hi @grosser,
Thanks for your reply and sorry about such interruption.

I just noticed that `cache:` was enabled for Travis builds in this pull request, whereas [Travis caching feature was available since December 17, 2014 (more than a year earlier)](https://blog.travis-ci.com/2014-12-17-faster-builds-with-container-based-infrastructure). I am interested in understanding the reason behind delaying the cache feature.

Thank you.

**Comment by grosser** [comment]
because nobody reached out and told us to do it ... found it by accident and then started using it here when touching it next

**Comment by Taher-Ghaleb** [comment]
I got your point. Thank you so much, @grosser, for your gentle responsiveness.

**Pull request merged**